isis/src/base/objs/CameraFactory/CameraFactory.cpp

```diff
@@ -128,39 +130,49 @@ namespace Isis {
     }
   }
 
+  
+  /**
+   * Just initialize the CSM and ISIS plugin lists.
+   */
+  void CameraFactory::Create() {
```

Comment:
Calling Create from a factory that neither takes any data in or returns anything does not make sense to me. I would much rather this say something like initializePlugins.

Reply:
Another option would be to make initPlugin public as it's not safe to call multiple times.

Reply:
Maybe a little too much future thinking. This class should be a singleton and not a collection of static functions. We'll fix it.